presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoStatement.java

```diff
@@ -194,6 +196,15 @@ public void setCursorName(String name)
         // ignore: positioned modifications not supported
     }
 
+    private Map<String, String> getStatementSessionProperties()
+    {
+        ImmutableMap.Builder<String, String> sessionProperties = ImmutableMap.builder();
+        if (queryTimeoutSeconds.get() > 0) {
+            sessionProperties.put("query_max_run_time", queryTimeoutSeconds.longValue() + "s");
```

Comment:
This is an `AtomicInteger`, so we can use `get()` here (no need to convert to long)